futures-util/src/future/mod.rs

```diff
@@ -714,6 +716,35 @@ pub trait FutureExt: Future {
         assert_future::<Self::Item, Self::Error, _>(inspect::new(self, f))
     }
 
+    /// Do something with the error of a future, passing it on.
+    ///
+    /// When using futures, you'll often chain several of them together.
+    /// While working on such code, you might want to check out what's happening
+    /// to the errors at various parts in the pipeline. To do that, insert a
+    /// call to `inspect_err`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate futures;
+    /// # extern crate futures_executor;
+    /// use futures::prelude::*;
+    /// use futures::future;
+    /// use futures_executor::block_on;
```

Comment:
As in the other change, I'd use `futures::executor` rather than `futures_executor`.